src/vmm/src/guest_config/aarch64/static_cpu_templates/mod.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+use versionize::{VersionMap, Versionize, VersionizeError, VersionizeResult};
+use versionize_derive::Versionize;
+
+/// Module with V1N1 CPU template for aarch64
+pub mod v1n1;
+
+/// Template types available for configuring the ARM CPU features that map
+/// to EC2 instances.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Serialize, Deserialize, Versionize)]
+pub enum Aarch64StaticCpuTemplates {
+    // Needed for compatibility
+    /// Empty0
+    Empty0,
```

Comment:
We need these dummy values to make `StaticTemplate` for aarch64 enum be like x86 version, because in the snapshots we store this value. And for x86 the None value is 4th, so needed some padding.